python/cuml/cuml/accel/tests/scikit-learn/xfail-list.yaml

```diff
@@ -1399,6 +1396,8 @@
   - "sklearn.tests.test_common::test_estimators[TruncatedSVD()-check_n_features_in_after_fitting]"
   - "sklearn.tests.test_common::test_estimators[SVC()-check_n_features_in_after_fitting]"
   - "sklearn.tests.test_common::test_estimators[SVR()-check_n_features_in_after_fitting]"
+  - "sklearn.tests.test_common::test_estimators[KNeighborsClassifier()-check_n_features_in_after_fitting]"
+  - "sklearn.tests.test_common::test_estimators[KNeighborsRegressor()-check_n_features_in_after_fitting]"
```

Comment:
I'm aware that this is not formalized (yet), but if we maintain alphabetical order, we can avoid this diff, right?

Reply:
I'm not sure I follow? This is a group of tests that fail for the same reason.

Reply:
The current list is (should?) be mostly alphabetically ordered. The diff makes it look like those specific entries were just moved from a different location.

Reply:
I guess I just misread the diff. Never mind, then.